src/vmm/src/persist.rs

```diff
@@ -524,14 +500,73 @@ pub enum GuestMemoryFromUffdError {
     Send(#[from] vmm_sys_util::errno::Error),
 }
 
-fn guest_memory_from_uffd(
+// TODO remove these when the UFFD crate supports minor faults for guest_memfd
+const UFFDIO_REGISTER_MODE_MINOR: u64 = 1 << 2;
+const UFFDIO_REGISTER: u64 = 0xc020aa00;
```

Comment:
Can drop it once the UFFD create supports minor mode in general, right? This is not specific to guest_memfd if I'm understanding correctly, so theoretically we could already open a PR over there to add support for this, right?

same for uffdio_continue actually

Reply:
For later reference, I've opened https://github.com/bytecodealliance/userfaultfd-rs/pull/72 for the minor fault API